src/DynamoCoreWpf/Views/Preview/Watch3DView.xaml

```diff
@@ -122,16 +123,7 @@
                 <MouseBinding Command="hx:ViewportCommands.SetTarget" Gesture="Ctrl+RightDoubleClick"/>
                 <MouseBinding Command="hx:ViewportCommands.Reset" Gesture="Ctrl+MiddleDoubleClick"/>
                 <MouseBinding Command="{Binding LeftClickCommand}" Gesture="LeftClick"/>
-            </hx:Viewport3DX.InputBindings>
-            <hx:DirectionalLight3D Name="key" Color="{Binding DirectionalLightColor}" Direction = "{Binding DirectionalLightDirection}"/>
```

Comment:
All of these `GeometryModel3D` objects are no longer required, because we are creating them on the fly.